lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -190,8 +207,8 @@ def compare_results(
             (None, {}, ["fio_1", "fio_2"]),
             (None, {"access": "public"}, ["fio_1", "fio_2"]),
             ("drb", {"name": "fio"}, ["fio_1", "fio_2"]),
-            ("drb", {"name": "fio", "limit": 1}, ["fio_1", "fio_2"]),
-            ("drb", {"name": "fio", "limit": 1, "offset": 2}, ["fio_1", "fio_2"]),
+            ("drb", {"name": "fio", "limit": "1"}, ["fio_1", "fio_2"]),
+            ("drb", {"name": "fio", "limit": 1, "offset": "2"}, ["fio_1", "fio_2"]),
             ("drb", {"name": "fio", "offset": 1}, ["fio_1", "fio_2"]),
             ("drb", {"name": "fio", "offset": 2}, ["fio_1", "fio_2"]),
```

Comment:
I'm confused:  when are the values of `"offset"` and `"limit"` supposed to be strings and when are they supposed to be numbers?

Reply:
On input to the API they're always strings, because they're part of the URI. There's no way to pass an integer, or any other type. On the other hand, internally, parameter normalization integerizes them; while on the other side `requests` (and I believe the Javascript equivalents) are perfectly capable of taking a JSON object with integers and stringifying them.

And mostly because these parametrizations had previously (if somewhat accidentally) taken advantage of that, I decided to test both cases.

Reply:
> I decided to test both cases.

OK, good.  A comment indicating that intentionality would be reassuring....  :-)